examples/ui-demo/src/app/state.tsx

```diff
@@ -104,6 +108,12 @@ export function ConfigContextProvider(props: PropsWithChildren) {
       sections.push([{ type: "passkey" }]);
     }
 
+    if (config.auth.showSocial && config.auth.addGoogleAuth) {
+      sections.push([
+        { type: "social", authProviderId: "google", mode: "popup" },
+      ]);
+    }
```

Comment:
And then here you would do something like

```suggestion
    if (config.auth.showSocial && config.auth.addGoogleAuth) {
      if (config.showPasskey) {
           sections.at(-1).push({ type: "social", authProviderId: "google", mode: "popup" });
       } else {
          sections.push([
            { type: "social", authProviderId: "google", mode: "popup" },
          ]);
       }
    }
```